### PR TITLE
Added another strategy for partials

### PR DESCRIPTION
**Pull request by KingWitherBrine**
Changed grammar a bit, and added another strategy.

### CODE REVIEW

File: content/1_General/Contest_Strategy.mdx [hunk 1]
```diff
@@ -27,22 +27,23 @@ Historically, the first platinum problem has never been the hardest (though you
...
+It's the worst feeling in the world to sink a couple hours into a problem before realizing that a different problem would've been easy pickings; we can easily avoid this situation with proper time allocation. You have to use your own judgement in deciding what is likely solvable and when you should quit. Generally, don't spend "too long" on one problem, and stay away from problems that look like they test something you don't know well.
 
 ### When you see a solution
 
 If you immediately get the solution to a problem while reading it, it might be a good idea to code it up quickly and get it out of the way.
 
 It also might be a good idea to make some progress on other problems before coding the one you know how to do. When you come back and start coding, you can make sure your solution still makes sense and then you have ideas from the other problems bouncing around in your head while you're coding.
 
-### When there are partials
+### When there are partial accepted answers
```
Comment (thecodingwizard) [inline content/1_General/Contest_Strategy.mdx:38]: Is "partials" better or "partial accepted answers"? @bqi343

File: content/1_General/Contest_Strategy.mdx [hunk 2]
```diff
@@ -27,22 +27,23 @@ Historically, the first platinum problem has never been the hardest (though you
...
 
 If you can get points from partially solving a problem (e.g. in USACO), then it's a good idea to keep those subtasks in mind. There are two main approaches to partially solving problems.
 
 - Use most of your time to think about the full problem, then code the partials if you haven't found the solution and your time will soon run out.
   - If you find the full solution, be certain your solution is correct, lest you waste an hour coding something completely wrong.
   - Judge the difficulty of the partials carefully! It could take anywhere from 30 seconds to 30 minutes to code a partial, or maybe a partial wasn't as trivial as you thought!
 - Work up from the partials, coding the easiest and then going to the hardest. The partial subtasks may help you think of the full solution, and will also help you stay on track.
+- Think about if you can quickly think of the cases you are missing, and if you don't, that's okay just move on.
```
Comment (thecodingwizard) [inline content/1_General/Contest_Strategy.mdx:46]: what does this mean? (Also it doesn't really seem like a "main approach" for partials, so perhaps it shouldn't be included in this list?)

File: content/1_General/Contest_Strategy.mdx (hunk 1 above)
Comment (bqi343) [reply content/1_General/Contest_Strategy.mdx:38]: I don't think "partial accepted answers" makes sense. "when there is partial scoring" or "when there are subtasks" maybe

Comment (KingWitherBrine) [reply content/1_General/Contest_Strategy.mdx:38]: Oh, I see now, that makes more sense. Thank you.

File: content/1_General/Contest_Strategy.mdx (hunk 2 above)
Comment (KingWitherBrine) [reply content/1_General/Contest_Strategy.mdx:46]: Sometimes when you are writing your code, you might miss some cases, but once you really think about it, you might be able to think of which cases you didn't think of.

**Pull request closed without merging**